botCommands/faq.js

```diff
@@ -22,9 +22,9 @@ const command = {
         }
       });
     } else if (mentionedUsers.length === 1) {
-      users = `, ${mentionedUsers[0]}`;
+      users = `${mentionedUsers[0]}`;
```

Comment:
The mentions should read a bit more smoothly, like the /time command. Here, they kind of just get tacked onto the beginning of the string, instead of reading like a regular sentence. I would see how the time command handles this and integrates the mention into the output.

Reply:
Also, there's a bit of whitespace at the beginning of the text when you have more than one mention.

Reply:
I moved the users mention to the start of the second sentence I think that flows better.